pkg/cri/sbserver/images/image_pull.go

```diff
@@ -328,6 +328,11 @@ func (c *CRIImageService) getLabels(ctx context.Context, name string) map[string
 // in containerd. If the reference is not managed by the cri plugin, the function also
 // generates necessary metadata for the image and make it managed.
 func (c *CRIImageService) UpdateImage(ctx context.Context, r string) error {
```

Comment:
would you mind to test with starting pod with `busybox` image? The image is without domain name and `library` namespace. I remembered that kubelet won't add the default domain name. So I want to confirm that it doesn't break anything. thanks

Reply:
it's ok, kubelet does not add the default domain prefix, but the `cri plugin` converts `busybox:latest` to `docker.io/library/busybox:latest`

After the pod starts, use crictl and ctr to view the image
```bash
$ ctr -n k8s.io images | grep busybox                                               root@caiwei-master:~# ctr -n k8s.io images ls | grep busybox
docker.io/library/busybox:latest                                                                                                        application/vnd.docker.distribution.manifest.list.v2+json sha256:3fbc632167424a6d997e74f52b878d7cc478225cffac6bc977eedfe51c7f4e79 2.1 MiB   linux/386,linux/amd64,linux/arm/v5,linux/arm/v6,linux/arm/v7,linux/arm64/v8,linux/mips64le,linux/ppc64le,linux/riscv64,linux/s390x io.cri-containerd.image=managed
docker.io/library/busybox@sha256:3fbc632167424a6d997e74f52b878d7cc478225cffac6bc977eedfe51c7f4e79                                       application/vnd.docker.distribution.manifest.list.v2+json sha256:3fbc632167424a6d997e74f52b878d7cc478225cffac6bc977eedfe51c7f4e79 2.1 MiB   linux/386,linux/amd64,linux/arm/v5,linux/arm/v6,linux/arm/v7,linux/arm64/v8,linux/mips64le,linux/ppc64le,linux/riscv64,linux/s390x io.cri-containerd.image=managed
$ crictl images | grep busybox
docker.io/library/busybox                                         latest              a416a98b71e22       2.22MB
```